app/views/drinks/show.html.erb

```diff
@@ -1,6 +1,14 @@
 <h1> <%= @drink.name %> </h1>
 <div>
-  <%= @drink.user.email %>
+  Created by: <%= @drink.user.email %>
+</div>
+<div>
+  <% if @drink.alcoholic %>
```

Comment:
idiomatically we infer that a method returns true or false by ending it with a `?`

For boolean fields in rails, the framework creates this method for us automagically, so consider:

```erb
<% if @drink.alcoholic? %>
```